package.json

```diff
@@ -10,8 +10,10 @@
   "dependencies": {
     "backo2": "^1.0.2",
     "graphql-subscriptions": "^0.2.0",
```

Comment:
Ok, so while we're here, can you move this one in `devDependencies` and `peerDependencies`.